src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -49,6 +49,7 @@ export class CommentEditorComponent implements OnInit {
 
   dragActiveCounter = 0;
   uploadErrorMessage: string;
+  @Input() maxLength = 40000;
```

Comment:
Let's remove magic numbers in our code by extracting them out as constants (refer to line 9 - 16).

Reply:
Thank you for the comment! I have made the changes, please help to see if it is okay now 😄